docs/dev/datastore.rst

```diff
@@ -50,6 +50,7 @@ your ``$PUDL_INPUT`` directory, organized by data source, form, and DOI::
     data/ferc2/
     data/ferc60/
     data/ferc714/
+    data/phmsagas/
```

Comment:
This is how the dataset was named when we first added it to PUDL. I'm not sure it makes the most sense (vs. just `phmsa`) but we'd have to overhaul all of our names in two different repos to change it at this point.

Reply:
That does sound annoying, but also `phmsagas` is also annoying?? @cmgosnell what do you think? Who did the original naming?

Reply:
As per discussion in slack, this reflects the desire to name our sources {distributor}{source}, as we do for `eia923` or `epacems`. Rather than splitting it into `phmsatransmission`, `phmsadistribution`, we're following the high-level split between gas and hazardous materials data reported by PHMSA.